pkg/config/evaluation_test.go

```diff
@@ -137,6 +138,87 @@ func TestEvaluateConditionsV1(t *testing.T) {
 
 }
 
+func TestNotAvtiveEvaluateConditionsV1(t *testing.T) {
+	filesMock := mock.FilesMock{}
+
+	runConfig := RunConfigV1{
+		PipelineConfig: PipelineDefinitionV1{
+			Spec: Spec{
+				Stages: []Stage{
+					{
+						Name:        "stage1",
+						DisplayName: "Test Stage 1",
+						Steps: []Step{
+							{
+								Name:       "step1_1",
+								Conditions: []StepCondition{},
+								NotActiveConditions: []StepCondition{
+									{ConfigKey: "testKeyNotExisting"},
+									{ConfigKey: "testKey"},
+								},
+								Orchestrators: []string{"Jenkins"},
+							},
+							{
+								Name: "step1_2",
+								Conditions: []StepCondition{
+									{ConfigKey: "testKey"},
+								},
+								NotActiveConditions: []StepCondition{
+									{ConfigKey: "testKeyNotExisting"},
+								},
+							},
+							{
+								Name:                "step1_3",
+								Conditions:          []StepCondition{},
+								NotActiveConditions: []StepCondition{},
+							},
+						},
+					},
+					{
+						Name:        "stage2",
+						DisplayName: "Test Stage 2",
+						Steps: []Step{
+							{
+								Name: "step2_1",
+								Conditions: []StepCondition{
+									{ConfigKey: "testKeyNotExisting"},
+									{ConfigKey: "testKey"},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	config := Config{Stages: map[string]map[string]interface{}{
```

Comment:
test also explicit activation/deactivation of stage. This should not be impacted - at least in the mid term ...

Reply:
currently if a single step return true then the stage is activated , we can remove this when we look into stage activation / deactivation

Reply:
added a negative stage test in the last push